### PR TITLE
RS: Update mac-os.md grammatical error 

### PR DESCRIPTION
**Pull request by nimakarimiank**
grammatical issue

### CODE REVIEW

**Review by dwdougherty — COMMENTED** [review]
Thank you, Nima! I think that entire paragraph needs some work, so let's park this PR for now and I'll return to it later when I have some more time consider it more carefully.

**Comment by nimakarimiank** [comment]
@dwdougherty I didn't review the file thoroughly. I can do it and make another commit for you to review 🙏🏼

**Comment by dwdougherty** [comment]
@nimakarimiank No need. I'll take a look once I get on the other side of some release activity work.

File: content/operate/oss_and_stack/install/install-stack/mac-os.md [hunk 1]
```diff
@@ -36,7 +36,7 @@ The `redis-stack-server` cask will install all Redis and Redis Stack binaries. H
 
 ### First-time Redis installation
 
-If this is the first time you've installed Redis on your system, then all Redis Stack binaries be installed and accessible from the `$PATH`. On Apple silicon Macs, this assumes that `/opt/homebrew/bin` is in your path. On Intel-based Macs, `/usr/local/bin` should be in the `$PATH`.
+If this is the first time you've installed Redis on your system, then all Redis Stack binaries are installed and accessible from the `$PATH`. On Apple silicon Macs, this assumes that `/opt/homebrew/bin` is in your path. On Intel-based Macs, `/usr/local/bin` should be in the `$PATH`.
```
Comment (dwdougherty) [inline content/operate/oss_and_stack/install/install-stack/mac-os.md:39]: I think this is better. What do you think Nima?

```suggestion
If this is the first time you've installed Redis on your system, you need to be sure that your `PATH` variable includes the Redis Stack installation location. This location is either `/opt/homebrew/bin` for Apple silicon Macs or `/usr/local/bin` for Intel-based Macs.
```

**Review by dwdougherty — CHANGES_REQUESTED** [review]
See suggestion.

**Comment by nimakarimiank** [comment]
This looks good to me ☺️

**Comment by dwdougherty** [comment]
H Nima. Thanks. But it looks like you resolved without committing the suggestion. Can you go back in and commit it?

File: content/operate/oss_and_stack/install/install-stack/mac-os.md (hunk 1 above)
Comment (nimakarimiank) [reply content/operate/oss_and_stack/install/install-stack/mac-os.md]: lgtm

**Review by nimakarimiank — COMMENTED** [review]
lgtm

**Comment by nimakarimiank** [comment]
@dwdougherty I was using Gits mobile app I thought it would do it for me since it didn't have the option to do so.
sorry!

**Review by dwdougherty — APPROVED** [review]
LGTM. Thanks again, Nima!

**Pull request merged**